blockchain/spam_throttler.go

```diff
@@ -107,18 +110,19 @@ func (t *throttler) adjustThreshold(ratio uint) {
 	var newThreshold int
 	// Decrease threshold if a fail ratio is bigger than target value to put more addresses in throttled map
 	if ratio > t.config.TargetFailRatio {
-		newThreshold = t.threshold - t.config.IncreaseWeight
-
-		// Set minimum threshold
-		if newThreshold < t.config.IncreaseWeight {
-			newThreshold = t.config.IncreaseWeight
+		if t.threshold-t.config.ThresholdAdjustment > t.config.MinimumThreshold {
+			newThreshold = t.threshold - t.config.ThresholdAdjustment
+		} else {
+			// Set minimum threshold
+			newThreshold = t.config.MinimumThreshold
```

Comment:
This value is changed from 5 to 100 . Is it ok?

Reply:
Yes. it's intended. the minimum threshold is changed from IncreaseWeight (5) to MinimumThreshold (300)